events/reporter.go

```diff
@@ -231,10 +236,9 @@ func ReportNodeStatusUpdate() {
 			select {
 			case sub <- struct{}{}:
 			default:
-				log.Debug("reporter would block on subscriber %v", sub)
+				log.Debug("reporter would block on subscriber")
 			}
 		}
-		log.Info("reported status update to subscribers: %v")
```

Comment:
why remove this one?

Reply:
It had no information associated with it, because the status update is just a `struct{}{}`. I think this was in response to a previous PR comment